config/locales/hyrax.en.yml

```diff
@@ -1132,17 +1142,21 @@ en:
       defaults:
         admin_set_id: Administrative Set
         based_near: Location
+        contributor: Contributor
         creator: Creator
         date_created: Date Created
-        description: Description
+        description: Abstract or Summary
         embargo_release_date: until
         files: Upload a file
+        identifier: Identifier
         keyword: Keyword
         lease_expiration_date: until
         license: License
         member_of_collection_ids: Add to collection
+        publisher: Publisher
         related_url: Related URL
         rights_statement: Rights statement
+        source: Source
```

Comment:
remove line 1159 for this and all other languages

Reply:
Line 1159 deleted